### PR TITLE
Add export to ESRI shapefiles for gebieden

### PR DESCRIPTION
**Pull request by jasper-jager**
Added an ESRI shape exporter and added the option to upload extra files which are created during and export product

### CODE REVIEW

**Comment by jasper-jager** [comment]
The docker container was missing the correct gdal binding, so I will fix this first

File: src/tests/exporter/test_export.py [hunk 1]
```diff
@@ -71,3 +76,22 @@ def test_export_to_file(monkeypatch):
...
+
+    file_name = 'esri.shp'
+
+    # Update records to contain an geometry collection
+    records = [{'identificatie': '2', 'geometrie': {'type': 'GeometryCollection', 'geometries': [{'type': 'LineString', 'coordinates': [[125891.16, 480253.38], [125891.07, 480253.34]]}, {'type': 'Polygon', 'coordinates': [[[125891.16, 480253.38], [125893.06, 480250.0], [125892.57, 480250.0]]]}]}}]
+
+    export_to_file(catalogue, collection, esri_exporter, file_name, format)
+
+    # Remove created files
+    for file in ['esri.shp', 'esri.dbf', 'esri.shx', 'esri.prj']:
```
[the remark this answers is not in the document]
Comment (jasper-jager) [reply src/tests/exporter/test_export.py]: I've tried, but the gdal library is a python binding to a c library which doesn't seem to work. Do you have any suggestions?

File: src/gobexport/exporter/config/meetbouten.py [hunk 2]
```diff
@@ -17,6 +17,7 @@ class MeetboutExportConfig:
             'exporter': dat_exporter,
             'endpoint': '/gob/meetbouten/meetbouten/?view=enhanced',
             'filename': 'MBT_MEETBOUT.dat',
+            'mime_type': 'plain/text',
```
[the remark this answers is not in the document]
Comment (jasper-jager) [reply src/gobexport/exporter/config/meetbouten.py]: Python mimetypes lib is mainly used for looking up mimetypes based on extension. Can't seem to find a existing library with a list of mimetypes as a constants. Should we implement or own or just leave is as is for now?

File: src/tests/exporter/test_export.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Vegter) [reply src/tests/exporter/test_export.py]: OK, it's a GDAL internal, let's leave this for now.

**Pull request merged**